core/unix/memquery.c

```diff
@@ -193,6 +201,13 @@ memquery_library_bounds_by_iterator(const char *name, app_pc *start /*IN/OUT*/,
                         ASSERT_NOT_REACHED();
                     }
                 } else {
+#ifdef STANDALONE_UNIT_TEST
+                    /* For the unit test, it's more convenient to return an
+                     * incorrect result here to fail the specific query, rather
+                     * than crashing the entire thing.
```

Comment:
The crashes came from my incorrect test results (related to the old 64-bit assumption in `record_memquery_library_bounds_by_iterator`), I do not have a legitimate map file which actually causes this.

I guess given that we should probably revert this change so that we don't confuse future readers about the possibility of this assert being something that actually triggers in real applications. wdyt?

Reply:
Yes, I would revert.

Reply:
Done.